src/js/flows/refreshConnectionsSpaceNames.ts

```diff
@@ -0,0 +1,18 @@
+import {Thunk} from "../state/types"
+import Clusters from "../state/Clusters"
+import Spaces from "../state/Spaces"
+
+export default function refreshConnectionsSpaceNames(): Thunk<Promise<void[]>> {
```

Comment:
Since we are refreshing all the data about the space and not just the names anymore, a better name might be "refreshSpacesForAllConnections" ?